docs/questions-and-answers.md

```diff
@@ -1,48 +1,39 @@
 <details><summary>What is an AEA?</summary>
-AEA stands for Autonomous Economic Agent. AEAs act independently of constant user input and autonomously execute actions to achieve their goals. Their purpose is to create economic value for you, their owner.
-<br><br>
```

Comment:
When formatting is in place, I recommend not changing it. Everyone has different tastes. As long as it is consistent we don't need to change it every time someone else reviews it.

Reply:
ok I reverted the new lines across because I agree with your comment. 

I kept the one above because I removed the "link without a link" statement, which I find inconsistent.